core/common/tarutil.hpp

```diff
@@ -16,9 +17,15 @@ namespace fc::common {
   outcome::result<void> zipTar(const std::string &input_path,
                                const std::string &output_path);
 
```

Comment:
```suggestion
  outcome::result<void> zipTar(const std::string &input_path,
                               const std::string &output_path);

```
`boost::filesystem::path` contains `std::string`, can remove string overload